client/src/AppBundle/Entity/Report/Satisfaction.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace AppBundle\Entity\Report;
+
+use DateTime;
+use JMS\Serializer\Annotation as JMS;
+
+class Satisfaction
+{
+    /**
+     * @JMS\Type("integer")
+     */
+    private $id;
+
+    /**
+     * @JMS\Type("integer")
+     */
+    private $score;
+
+    /**
+     * @JMS\Type("string")
+     */
+    private $comments;
+
+    /**
+     * @JMS\Type("string")
+     */
+    private $deputyrole;
+
+    /**
+     * @JMS\Type("string")
+     */
+    private $reporttype;
+
+    /**
+     * @JMS\Type("string")
```

Comment:
I'd suggest this to be `DateTime` so we are in control of how we format it in the CSV.